src/script/build-with-container.py

```diff
@@ -95,6 +94,12 @@ def __str__(self):
             return self.value
 
 
+try:
+    from functools import cache as ftcache
```

Comment:
1) supposedly cache is in 3.9; are we trying to support pythons even older?
2) lrucache has been there for a long time; wouldn't we prefer to fall back to it rather than no caching at all?

Reply:
1. I tested with 3.8
2. I was being lazy and going fast and decided to punish people running old versions ;-)